akka-stream-testkit/src/main/scala/akka/stream/testkit/javadsl/TestKit.scala

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (C) 2018 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.stream.testkit.javadsl
+
+import akka.stream.Materializer
+import akka.stream.impl.PhasedFusingActorMaterializer
+import akka.stream.testkit.scaladsl
+
+object TestKit {
```

Comment:
If in Java and using both `akka.testkit.Testkit` and needing to use this one it would be a bit of an import-pains... Would we consider calling this `StreamTestKit`? (as we have done in the past internally)

Reply:
I'll fix it up. Thanks.